src/config/constants.ts

```diff
@@ -45,6 +45,11 @@ export const Routes = {
     DEPLOYMENT_TEMPLATE: 'app/template',
     DEPLOYMENT_TEMPLATE_UPDATE: 'app/template/update',
 
+    DEPLOYMENT_MANIFEST: 'app/template/data',
+    DEPLOYMENT_OPTIONS:'app/template/list',
+    DEPLOYMENT_TEMPLATE_NEW: 'app/template/data',
```

Comment:
Why are we creating 2 diff constant for same value?

Reply:
resolved